public/buyer_assets/js/main.js

```diff
@@ -470,8 +470,12 @@ Note: main.js, All Default Scripting Languages For This Theme Included In This F
  $(".qtybutton").on("click", function() {
     var $button = $(this);
     var oldValue = $button.parent().find("input").val();
+	var maxValue = $button.parent().find("input").attr('max');

```

Comment:
alignment is not correct here on line 473 and 476 - 478.

Reply:
please remove commented code on line number 190 -194.

Reply:
done